### PR TITLE
Add styling pull request

### PR DESCRIPTION
**Pull request by alicemirigo92**
1. Understand the concept of state management.
2. Use store, actions and reducers in React.
3. Connect React and Redux.
4. Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/styles/Navigation.css [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  justify-content: space-between;
   align-items: center;
-  padding: 2rem;
+  justify-content: space-between;
   background-color: #fff;
-  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  margin: 5% 5%;
+  padding: 0 40px;
+  box-shadow: 0 0 10px 5px rgba(238, 239, 243, 0.8);
 }
```
Comment (amejid) [inline src/styles/Navigation.css:11]: - [ ] Great job with the layout. :+1: Let's work on some more details to match the given design. 
  - Kindly add vertical spacing on the navigation bar. 😎️
  - Please use the same `font-family` as the design, the design is using `Montserrat` in the navigation bar. 🤝️
  - Kindly use uppercase letters for the navigation links. 🥸️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: src/styles/BookList.css [hunk 2]
```diff
@@ -25,12 +27,13 @@
 }
 
 .list-item {
```
Comment (amejid) [inline src/styles/BookList.css:23]: - [ ] Let's work on some details in the book details section. :+1: 
  - Kindly make sure the book category(i.e. Fiction) is using `Montserrat` font family. 🧐️
  - Every other text in this section is using the `RobotoSlab` font family.  ✍️ 
  - Please remove the filling on the remove button to match the design. 🤓️
  - Please make sure you are using the correct color for the text. :+1: 

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: src/styles/BookList.css [hunk 3]
```diff
@@ -25,12 +27,13 @@
...
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 0 15px 10px rgba(238, 239, 243, 0.8);
   display: flex;
```
Comment (amejid) [inline src/styles/BookList.css:28]: - [ ] Kindly use the correct color on the status circle. The remaining area on the circle is gray in the design. Please do the same thing to match the design. Kindly use the correct font family which is `Montserrat`. I suggest you minimize the size of the circle, it is looking a bit bigger when compared to the design. Please use the correct color for the text too, it is looking darker on your design. 🥸️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: src/styles/BookList.css [hunk 4]
```diff
@@ -25,12 +27,13 @@
...
-  justify-content: space-between;
-  padding: 40px;
-  background-color: white;
-  border: 0.5px solid rgb(210, 181, 181);
-  margin: 11px 0;
+  gap: 10%;
```
Comment (amejid) [inline src/styles/BookList.css]: - [ ] Kindly use the correct font family which is `RobotoSlab` for this section. 🤝️
- [ ] Please put the texts `CURRENT CHAPTER` and `UPDATE PROGRESS` on a single line as per the design requirements. 🥸️
- [ ] Kindly check the color you are using for the text `CURRENT CHAPTER` 🙂️

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: src/styles/BookList.css [hunk 5]
```diff
@@ -25,12 +27,13 @@
...
+  margin-bottom: 20px;
+  padding: 15px 20% 15px 45px;
 }
```
Comment (amejid) [inline src/styles/BookList.css:33]: - [ ] Kindly align the text `ADD NEW BOOK` to the left as per the design. Please also make sure you are using the correct color for the text. 🥸️

**Your design**
Screenshot from 2023-03-11 11-41-44

**Zeplin design**
Screenshot from 2023-03-11 11-41-34

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @alicemirigo92, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Good layout :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- checking the remove functionality, I don't think it is working right now.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/Books.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  margin-top: 50px;
+  background-color: #fafafa;
+}
+
+.header {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 95%;
+}
```
Comment (CollinsTatang) [inline src/styles/Books.css]: - [ ] Please fix your font and button style it doesn't match that of the [Zeplin](https://zeplin.io/) design as requested by the previous reviewer.
**Zeplin Design**
zeplin
**Your Design**
fix

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/Books.css (hunk 6 above)
Comment (alicemirigo92) [reply src/styles/Books.css]: hey @CollinsTatang  thanks for the review, but I think we have different screen resolutions, this is a video showing my project on my screen. https://drive.google.com/file/d/1Aa9D4JkZMLP6yucTH8LKZhVMFaDMv5Mk/view?usp=sharing

Comment (CollinsTatang) [reply src/styles/Books.css]: Ok @alicemirigo92 
I see from the video you are right 

- [ ] Please correct the remove button to match the design

**Review by rotimiazeez — APPROVED** [review]
Hi @alicemirigo92 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👇🏽
- Good communication with the CR 👍🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**